publication/src/main/scala/hmda/publication/reports/util/LoanTypeUtil.scala

```diff
@@ -0,0 +1,74 @@
+package hmda.publication.reports.util
+
+import akka.NotUsed
+import akka.stream.scaladsl.Source
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.publication.reports._
+import hmda.util.SourceUtils
+
+import scala.concurrent.Future
+
+object LoanTypeUtil extends SourceUtils {
+  def loanTypes[ec: EC, mat: MAT, as: AS](larSource: Source[LoanApplicationRegister, NotUsed]): Future[String] = {
+    for {
+      conv <- purposesOutput(larSource.filter(lar => lar.loan.loanType == 1))
+      fha <- purposesOutput(larSource.filter(lar => lar.loan.loanType == 2))
+      va <- purposesOutput(larSource.filter(lar => lar.loan.loanType == 3))
+      fsa <- purposesOutput(larSource.filter(lar => lar.loan.loanType == 4))
+    } yield {
+      s"""
+         |[
+         |  {
+         |    "loantype": "Conventional",
```

Comment:
Make sure to use the updated versions of the json files, since it makes the formatting consistent and correct for all of the reports. A1 is [here](https://github.com/schbetsy/hmda-viz-prototype/blob/ab3835eb3c14e684982831840e137e9ee7a69394/2017-json-formats/aggregate/A-1.json). The whole collection is [in this PR](https://github.com/cfpb/hmda-viz-prototype/pull/193).

Specifically, we need to use camel case for the json keys: `loanType`, `loanTypes`, `firstLienCount`, `juniorLienCount`, and `noLienCount`.